README.md

```diff
@@ -2102,6 +2103,53 @@ The error message thrown by mailgun is passed to the response object when an ema
 Mailgun error codes can be found
 [here](https://documentation.mailgun.com/en/latest/api-sending.html#error-codes).
 
+## Localization
+
+### Setting application properties
+
+Form flow library will read the languages set in an apps `application.properties`.
+
+```yaml
+# application.yaml
+form-flow:
+  languages: en, es
+```
+
+Our language selector menu button will create links both inside of the button and nav links based on
+the value provided in `form-flow.languages`. If none is given, the default is: `en`.
+
+#### Language tag syntax
+
+Refer to
+the [language tag syntax](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang#language_tag_syntax)
+to see what language you want to support.
+
+### Supplying translation files
+
+Translations need to be added to `messages_xx.properties` files in the `src/main/resources`
+directory, where `xx` is the language the translations are for.
+
+#### Messages in the library
+
+The library comes with [English](src/main/resources/messages-form-flow.properties)
+and [Spanish](src/main/resources/messages-form-flow_es.properties) messages. These load into the
+app messages and get override by message keys that exist in
```

Comment:
here's a spot where "app" being "application's" would parse more easily (at least for my brain).

Reply:
Actually, in reading this more, it talks about the starter app and how it's configured.  It would be good to use that as an example, but maybe more this way: 

`These are loaded into the application's messages and can get overridden by message keys that exist in the application's message files.  Here's an example of how our Starter Application overrides some message strings...`

Reply:
I can't point to can example in the starter app, because we purposefully don't override library messages right now. But I added a sentence regarding this.